sketches/src/main/java/org/apache/apex/malhar/sketches/QuantilesEstimator.java

```diff
@@ -0,0 +1,182 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.apex.malhar.sketches;
+
+import com.yahoo.sketches.quantiles.QuantilesSketch;
+
+import com.datatorrent.api.DefaultInputPort;
+import com.datatorrent.api.DefaultOutputPort;
+import com.datatorrent.api.annotation.OperatorAnnotation;
+import com.datatorrent.common.util.BaseOperator;
+
+/**
+ * An implementation of BaseOperator that computes a "sketch" (a representation of the probability distribution using
+ * a low memory footprint) of the incoming numeric data, and evaluates/outputs the cumulative distribution function and
+ * quantiles of the probability distribution. Leverages the quantiles sketch implementation from the Yahoo Datasketches
+ * Library.
+ * <p/>
+ * <b>Input Port(s) : </b><br>
+ * <b>data : </b> Data values input port. <br>
+ * <br>
+ * <b>Output Port(s) : </b> <br>
+ * <b>cdfOutput : </b>cumulative distribution function output port. <br>
+ * <b>quantilesOutput : </b>quantiles output port. <br>
+ * <br>
+ * <b>Partitions : No</b>, no will yield wrong results. <br>
```

Comment:
Please elaborate on "will yield wrong results," it would be helpful in guiding someone to fix this later.  .